Formula/awscli.rb

```diff
@@ -22,10 +22,6 @@ class Awscli < Formula
   depends_on "python@3.10"
   depends_on "six"
 
-  on_system :linux, macos: :ventura_or_newer do
-    depends_on "groff"
-  end
-
```

Comment:
Replace this with `uses_from_macos "mandoc"`.

Reply:
thank you!  i was trying to figure out how to get the ci to pass on ubuntu...